src/torrent/chunk_manager.h

```diff
@@ -105,9 +107,6 @@ class LIBTORRENT_EXPORT ChunkManager : private std::vector<ChunkList*> {
   void                inc_stats_not_preloaded()                 { m_statsNotPreloaded++; }
 
 private:
-  ChunkManager(const ChunkManager&) = delete;
-  ChunkManager& operator=(const ChunkManager&) = delete;
```

Comment:
I prefer deleted copy methods to be in private, it makes the code cleaner.

Reply:
It makes it more confusing to me. Applied anyways.